style.css

```diff
@@ -0,0 +1,1453 @@
+@import url("https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap");
+
+:root {
+  --font-fam: "Rubik", sans-serif;
+  --primary-color: white;
+  --secondary-color: #4caf50;
+  --tertiary-color: #333;
+  --button-background: #393e46;
+  --red-accent: #fd7014;
+  --greyish: grey;
+  --black-color: black;
+}
+
+body {
+  background-color: var(--tertiary-color);
+}
+ul.topnav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: var(--tertiary-color);
+  width: 100%;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+}
+
+ul.topnav li a {
+  display: block;
+  color: var(--primary-color);
+  text-align: center;
+  padding: 1.8rem;
+  text-decoration: none;
+}
+
+ul.topnav li a:hover:not(.active) {
+  background-color: var(--secondary-color);
+}
+
+ul.topnav li a.active {
+  background-color: var(--secondary-color);
+}
+
+ul.topnav li.right {
+  float: right;
+  font-family: var(--font-fam);
+  font-size: 1.5rem;
+  font-weight: 400;
+}
+
+ul.topnav li.left {
+  float: left;
+  font-size: 1.8rem;
+  margin-left: 3.5rem;
+  font-weight: bolder;
+  font-family: var(--font-fam);
+}
+@media screen and (max-width: 600px) {
```

Comment:
add media queries at the bottom of the file. so that its styling does not override.